### DISCUSSION TITLE
Will these Docker images be sunset?

### DISCUSSION THREAD

**Discussion by temmeand** [inventree/InvenTree#4496] [General]
Is InvenTree impacted by Docker's recent announcement about removing/sunsetting accounts in mid-April?

See https://github.com/docker/hub-feedback/issues/2314
or lots of news stories, e.g. https://noted.lol/docker-hub-deletion/

**Comment by SchrodingersGat** [comment]
@temmeand thanks for raising this, I hadn't seen it. Very frustrating!

The inventree docker images are hosted on a "user" account, which (I *think*) is not going to change?

@matmair what's your take on this?

**Reply by matmair** [reply]
I think there is no immediate need for action but in the long run, I would prefer GHCR as that would put everything in one place.
I also like how Github does 2FA better than Docker (the company). My faith in Docker (the company) being around long-term is limited - we could start pushing to both GHCR and DockerHub for redundancy's sake. If you want I can make a PR for it.

**Reply by SchrodingersGat** [reply]
A PR would be appreciated. I think it's a worthwhile path even if we do both in parallel

**Comment by SchrodingersGat** [comment]
https://www.docker.com/blog/we-apologize-we-did-a-terrible-job-announcing-the-end-of-docker-free-teams/
